authentication/src/main/java/com/cgi/bonnie/authentication/auth/ApplicationUserService.java

```diff
@@ -0,0 +1,27 @@
+package com.cgi.bonnie.authentication.auth;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ApplicationUserService implements UserDetailsService {
+
+    private final ApplicationUserDao applicationUserDao;
+
+    @Autowired
+    public ApplicationUserService(ApplicationUserDao applicationUserDao) {
+        this.applicationUserDao = applicationUserDao;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        return applicationUserDao.selectApplicationUserByUsername(username);
+    }
+
+    public UserDetails loadUserByEmail(String email) throws UsernameNotFoundException {
```

Comment:
IntelliJ says this method is not used

Reply:
Yes, this method is not used yet, because currently we are identifying the users based on username, but later on identifying them based on email address is going to be more useful. But I'm going to make a commit to identify them based on email address.